src/pages/admin/donate/components/generic-form/GenericForm.scss

```
@@ -30,7 +30,8 @@
             font-weight: 700;
             display: inline;
             cursor: pointer;
-            word-break: break-word;
+            overflow-wrap: anywhere;
```

Stylelint (17.9.0)

[error] Duplicate property "overflow-wrap" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)